templates/pro.html

```diff
@@ -0,0 +1,571 @@
+              <li class="p-list__item u-no-padding--bottom">FISMA</li>
+            </ul>
+          </div>
+          <div class="col-3">
+            <ul class="p-list--divided">
+              <li class="p-list__item u-no-padding--bottom">Center for Internet Security (CIS)</li>
+              <li class="p-list__item u-no-padding--bottom">Common Criteria</li>
+              <li class="p-list__item u-no-padding--bottom">PCI</li>
+              <li class="p-list__item u-no-padding--bottom">ISO 27001</li>
+              <li class="p-list__item u-no-padding--bottom">FIPS 140</li>
+              <li class="p-list__item u-no-padding--bottom">Cyber Essentials</li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="u-no-margin--bottom is-muted" />
+      <div class="col">
+        <div class="row">
+          <h2>チームに安心感のある運用を</h2>
+        </div>
+      </div>
+      <div class="col">
+        <div class="row">
+          <p>
+            業務に対応する安定性、セキュリティ、コンプライアンス。CVE露出期間を平均98日から1日に短縮。
+          </p>
+          <div class="p-section--shallow">
+            <div class="row">
+              <div class="col-2 p-heading--2">
+                <strong>1,800+</strong>
+              </div>
+              <div class="col-4">
+                <p>
+                  以上
+                  <br />
+                  高、危険レベルのパッチは
+                  <br />
+                  Ubuntu Proだけが提供
+                </p>
+              </div>
+            </div>
+            <hr class="u-no-margin--bottom is-muted" />
+            <div class="row">
+              <div class="col-2 p-heading--2">
+                <strong>10年間</strong>
+              </div>
+              <div class="col-4">
+                <p>
+                  メンテナンスは
+                  <br />
+                  スタック全体に対応
+                </p>
+              </div>
+            </div>
+            <hr class="u-no-margin--bottom is-muted" />
+            <div class="row">
+              <div class="col-2 p-heading--2">
+                <strong>24/7</strong>
+              </div>
+              <div class="col-4">
+                <p>
+                  時間年中無休
+                  <br />
+                  企業グレードの
+                  <br />
+                  サポート（オプション）
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="u-no-margin--bottom is-muted" />
+      <div class="col">
+        <h3 class="p-heading--2">
+          Ubuntu Proの特長
+        </h3>
+      </div>
+      <div class="col">
+        <p>CanonicalのUbuntu Proは、企業におけるUbuntu環境にサポートと保証を提供します。Ubuntu Proには、ツール、ビットストリーム、サポートサービスなどを含む多様なパッケージがあります。</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-start-large-4">
```

Comment:
I'm not sure if this is the best way to instantiate the columns, would appreciate a second opinion

Reply:
Made the changes in a [pastebin](https://pastebin.canonical.com/p/97rqX59S6n/) so it's a bit easier to follow, couple of best practices to note here: 
1. offsets still need defined cols, so `col-start-large-4` for example is incomplete without `col-9`, docs [here](https://vanillaframework.io/docs/patterns/grid#empty-columns)
2. we're still bound by a 12 col grid so if you have an offset like this one where you're starting after the 4th col that means you have 9 cols left which means the remaining nested cols have to add up to 9 (you currently have them set as col-3 and col-9 which breaks the gird) 
3. for some reason when we're nesting cols like this including the hr within the nested row breaks things so it has to be done before the nested row

Reply:
I was debating what would be the best way to arrange nested cols and the previous implementation with nested 12 grids were not showing the cols width as intended. I'll push up a change to use this approach, thank you!